.travis.yml

```diff
@@ -10,20 +11,23 @@ install:
     # Dependencies
     - sudo apt-get -qq update
     - sudo pip install --upgrade -qq pip
-    - sudo apt-get -qq install cdparanoia cdrdao flac gir1.2-glib-2.0 libcdio-dev libiso9660-dev libsndfile1-dev python-gi python-musicbrainzngs python-mutagen python-setuptools sox swig libcdio-utils
-    - sudo pip install pycdio==0.21 requests setuptools_scm
+    - sudo apt-get -qq install cdparanoia cdrdao flac libgirepository1.0-dev gir1.2-glib-2.0 libiso9660-dev libsndfile1-dev python-cddb sox swig libcdio-utils
 
+    # Travis' Ubuntu is very old, libcdio needs to be compiled
+    - wget https://ftp.gnu.org/gnu/libcdio/libcdio-2.0.0.tar.gz
+    - tar -xvzf libcdio-2.0.0.tar.gz
+    - pushd libcdio-2.0.0
+    - ./configure
+    - sudo make install
+    - popd
+    
     # Testing dependencies
-    - sudo apt-get -qq install python-twisted-core
-    - sudo pip install flake8
+    - sudo pip install .[test,lint]
 
     # Build bundled C utils
-    - cd src
+    - pushd src
     - sudo make install
-    - cd ..
-
-    # Installing
-    - sudo python setup.py install
```

Comment:
We probably still want to test whether installing works at all. We don't have a lot of functionality tests as it is (do we have any?), only unit tests, and this seems like a fairly basic one to have…

Reply:
This command is run by tox as part of the process it uses to run commands.

It generate a source distribution of the package, a virtual environment and then installs the package onto the virtual environment.

Excerpt from a tox command:

```
$ tox -e py27-lint
GLOB sdist-make: /home/pau/code/tui/whipper/setup.py
py27-lint create: /home/pau/code/tui/whipper/.tox/py27-lint
py27-lint inst: /home/pau/code/tui/whipper/.tox/.tmp/package/1/whipper-0.7.4.dev50+gcb3515c.zip
py27-lint installed: DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.,certifi==2018.11.29,chardet==3.0.4,configparser==3.7.1,entrypoints==0.3,enum34==1.1.6,flake8==3.7.5,functools32==3.2.3.post2,idna==2.8,mccabe==0.6.1,musicbrainzngs==0.6,mutagen==1.42.0,pycairo==1.18.0,pycdio==2.0.0,pycodestyle==2.5.0,pyflakes==2.1.0,PyGObject==3.30.4,requests==2.21.0,typing==3.6.6,urllib3==1.24.1,whipper==0.7.4.dev50+gcb3515c
```